Resources/Prototypes/_SpaceCats/Entities/Objects/Misc/pens.yml

```
@@ -0,0 +1,370 @@
+- type: entity
+  name: overpriced science pen
+  parent: Pen
+  id: PenScienceOverpriced
+  description: A pen you probably spent too much on. It isn't any different from the standart-issue, so what's wrong with the price anyway? At least the component quality is quite worth it.
+  components:
+  - type: Sprite
+    sprite: _SpaceCats/Objects/pens.rsi
+    state: scientist1
+  - type: Tool
+    qualities:
+      - Screwing
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)